src/app/(proper_react)/redesign/(public)/Faq.tsx

```diff
@@ -0,0 +1,140 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { ReactNode, useState } from "react";
+import { useL10n } from "../../../hooks/l10n";
+import styles from "./LandingView.module.scss";
+import { CloseBigIcon } from "../../../components/server/Icons";
+import { useTelemetry } from "../../../hooks/useTelemetry";
+
+export type FaqItemProps = {
+  question: string;
+  answer: string | ReactNode;
+  isExpanded: boolean;
+  onExpandAnswer: () => void;
+};
+
+const FaqItem = (props: FaqItemProps) => {
+  const { question, answer, isExpanded, onExpandAnswer } = props;
+
+  return (
+    <>
+      <dt className={styles.faqQuestion} onClick={onExpandAnswer}>
+        {question}
+        <CloseBigIcon alt="" className={`${isExpanded && styles.expanded}`} />
```

Comment:
It's currently impossible to bring focus to this element (and thus expand/collapse the entry) with the keyboard. [In Relay](https://github.com/mozilla/fx-private-relay/blob/6b0bcf7f17af81670a208160f1ed8645e15f2092/frontend/src/components/landing/FaqAccordion.tsx#L64) we fixed that by wrapping it in a button. If you don't like the extra wrapping, you could also use react-aria's `useButton` hook to turn `<dt>` into a button.

Reply:
Oh, and one thing I see we didn't do on Relay, but which would be good, is to add an [`aria-expanded`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded) and an `aria-controls` on the `<dt>`/`<button>`.

And since the docs say

> The accessible name of the controlling object should reflect this change.

I think that the most sensible way to do this is probably by adding an alt text to `<CloseBigIcon>` , something like `(Open)` / `(Closed)`, I suppose?

Reply:
The `<dl>`  is great though, btw!

Reply:
Or native collapse/expand using `<details>` and `<summary>` tags, although it might tweak the UI slightly.

```html
<details>
  <summary><h3>What kinds of websites sell my personal information?</h3></summary>
  <p>Certain websites are in the business of collecting and selling people’s personal information without their consent, which is unfortunately legal in the US. These sites are called data brokers and they make up a $240 billion dollar industry. They use sophisticated methods to collect personal, financial, location, and even health information, often without your consent or even your knowledge. They’ll sell what they’ve collected to third parties, profiting from your information and leaving you open to violations of your privacy and security.</p>
</details>
```

<details>
  <summary><h3>What kinds of websites sell my personal information?</h3></summary>
  <p>Certain websites are in the business of collecting and selling people’s personal information without their consent, which is unfortunately legal in the US. These sites are called data brokers and they make up a $240 billion dollar industry. They use sophisticated methods to collect personal, financial, location, and even health information, often without your consent or even your knowledge. They’ll sell what they’ve collected to third parties, profiting from your information and leaving you open to violations of your privacy and security.</p>
</details>